### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by Ocalak**
Fixing the typos and adding xlab, lab and legends for the plots.

### CODE REVIEW

File: 08-ch8.Rmd
Comment (mca91) [inline 08-ch8.Rmd:1]: Why did you add  a `?`  ?

**Comment by Ocalak** [comment]
Hey Martin, the equation (in line 260) is part of the question that starts in the line 258. That's why I added a "?". Sent from my iPhoneOn 29. Aug 2023, at 07:50, Martin C. Arnold ***@***.***> wrote:﻿
@mca91 commented on this pull request.



On 08-ch8.Rmd:
Why did you add  a ?  ?

**Comment by mca91** [comment]
Yeah, I did notice this at first. Thanks! :-)
